integration_tests/src/test/resources/xdist_it_log4j.properties

```diff
@@ -0,0 +1,24 @@
+log4j.appender.myConsoleAppender=org.apache.log4j.ConsoleAppender  
```

Comment:
this file needs copyright.  Ideally also description about what its for and maybe how its different from other log4j in tests directory